webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+import db.dataset
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/<uuid:job_id>/job-details", methods=["GET"])
```

Comment:
Yes I guess the job-id can be put in the end, I did this way to follow the datasets API.